.circleci/config.yml

```diff
@@ -654,3 +711,4 @@ workflows:
             - tests.regressions
             - tests.performance
             - tests.headers
+      - arm64_build
```

Comment:
I don't know if this works, but could this be another "top-level workflow" on the same level as `build-and-test`? Perhaps with the name `build-and-test-arm64` (or just `build-arm64` if we don't get the testing part yet in this PR).